common/sgx/verifier.c

```diff
@@ -325,6 +325,73 @@ static oe_result_t _fill_with_known_claims(
             sizeof(OE_CLAIM_VALIDITY_UNTIL),
```

Comment:
In line 242, `claims_length` check should include `OE_SGX_CLAIMS_COUNT`

Reply:
Thanks for point that out!